clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -74,6 +74,9 @@
                                                          + "elapses the client will resend the request if necessary or fail the request if "
                                                          + "retries are exhausted.";
 
+    public static final String INTERCEPTOR_CLASSES_CONFIG  = "interceptor.classes";
+    public static final String INTERCEPTOR_CLASSES_DOC = "A list of classes to use as interceptors. Implemeting the <code>ProducerInterceptor</code> and <code>ConsumerInterceptor</code> interfaces allows plugging in classes that can listen to events happening to the records. By default, there are no interceptors.";
```

Comment:
Maybe we could have separate doc strings for the consumer and producer? This all ends up in the config documentation on kafka.apache.org, so it might be a little confusing to mention ProducerInterceptor in the consumer's configuration.